server.js

```diff
@@ -1,35 +1,88 @@
-import express from 'express'
-import bodyParser from 'body-parser'
+import express, { response } from 'express'
 import cors from 'cors'
 import mongoose from 'mongoose'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
+import netflixData from './data/netflix-titles.json'
 
 const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
+const Serie= mongoose.model('Series', {
+  show_id: Number,
```

Comment:
Why do you need both show_id and _id?

Reply:
HI, @KarolCislowski I decided to use show_id that is the original id of the series as this data was converted from Json to mongo I didn't use the ID generated when it creates a new document in de DB because was too long and to understand better in this project I kept the original id.